extension/content/components/common/AppSidebar.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { Icon, Nav, Sidenav, Popover, Whisper } from "rsuite";
+
+import { useEnvironmentState } from "devtools/contexts/environment";
+import { splitCamelCase } from "devtools/utils/helpers";
+
+export const AppSidebar: React.FC = () => {
+  const { selectedKey } = useEnvironmentState();
+
+  const versionPopover = (
+    <Popover>
+      <dl>
+        {Object.entries(__BUILD__).map(([key, value]) => {
+          return (
+            <>
+              <dt>{splitCamelCase(key, { case: "title-case" })}</dt>
+              <dd>
+                {typeof value === "string" ? value : JSON.stringify(value)}
+              </dd>
+            </>
+          );
+        })}
+      </dl>
+    </Popover>
+  );
+
+  return (
+    <div className="app-sidebar">
+      <Sidenav className="h-100">
+        <Sidenav.Body className="d-flex flex-column h-100">
+          <Nav vertical className="flex-grow-1">
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="book" />}
+              to={`/${selectedKey}/recipes`}
+            >
+              Recipes
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="filter" />}
+              to={`/${selectedKey}/filters`}
+            >
+              Filters
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="table" />}
+              to={`/${selectedKey}/pref-studies`}
+            >
+              Pref Studies
+            </Nav.Item>
+            <Nav.Item
+              componentClass={NavLink}
+              icon={<Icon icon="puzzle-piece" />}
+              to={`/${selectedKey}/addon-studies`}
+            >
+              Add-on Studies
+            </Nav.Item>
+          </Nav>
+
+          <div className="text-subtle cursor-pointer p-2 flex-grow-0">
+            <Whisper
+              placement="topStart"
+              speaker={versionPopover}
+              trigger="click"
+            >
+              <button>{__BUILD__.version}</button>
```

Comment:
i think you should switch this to just a `span` to call less attention to this. i would also move `text-subtle` and `cursor-default` (instead of `cursor-pointer`) to this element instead of the containing `div`.

Reply:
How does making it a button call attention to it? It seems visually the same as a `button` or a `span` to me.

I suppose if it isn't a clickable expansion, and switches to an on-hover, then it makes sense to use span. I'd prefer to keep it as a click interaction though, and so marking it with a button element is more semantic.

Reply:
Moving the utility classes down to the deeper element makes sense though.

Reply:
> How does making it a button call attention to it? It seems visually the same as a button or a span to me.

Light theme:
<img width="167" alt="image" src="https://user-images.githubusercontent.com/987136/80743908-ce38d480-8aeb-11ea-8b78-d8687ee1e786.png">

Dark theme:
<img width="191" alt="image" src="https://user-images.githubusercontent.com/987136/80743848-b6615080-8aeb-11ea-98fc-e5c371bedb7f.png">

The button styling comes from the system right now so you might not be seeing it the same way.

As a `span` it looks like this for me (which I believe matches the screenshot you sent me from your system):
![image](https://user-images.githubusercontent.com/987136/80744082-1a841480-8aec-11ea-92c3-785b6cfe33b9.png)

Reply:
> I'd prefer to keep it as a click interaction though, and so marking it with a button element is more semantic.

My take on this is if you are providing additional contextual information this functions as a tooltip and therefore I expect it to show on hover. 

If it's a menu like in the case of the avatar in the top right corner when you're signed in and there's a caret icon suggesting dropdown then it could be triggered on click because it's clearer there is a click behavior available.

Reply:
That makes sense. I thought that the button styling would be coming from some reset somewhere, so I expected it to look the same on all platforms. I don't want to figure out cross platform theming of buttons, so lets go with the span and hover approach.